Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift

```diff
@@ -57,54 +57,78 @@ public class ResourcesProjectMapper: ProjectMapping {
             .appending(component: "Bundle+\(target.name).swift")
 
         let content: String = ResourcesProjectMapper.fileContent(targetName: target.name,
-                                                                 bundleName: bundleName)
+                                                                 bundleName: bundleName,
+                                                                 product: target.product)
         return (filePath, [.file(.init(path: filePath, contents: content.data(using: .utf8), state: .present))])
     }
 
-    static func fileContent(targetName: String, bundleName: String) -> String {
-        """
-        // swiftlint:disable all
-        import Foundation
+    static func fileContent(targetName: String, bundleName: String, product: Product) -> String {
```

Comment:
ah good catch! 

We were too caught up testing the static + bundle case 😅

We're missing a case here - `.staticFramework` targets 🤐, to be consistent, we can use the same check here that we use when we check if we need to generate a resource bundle target: `target.supportsResources`. 

Alternatively we can pass a boolean / enum (e.g. `accessorStyle: .bundle`, `.framework`) to explicitly state which style we need as we only need the the `.bundle` case in the case we automatically create the resources targets